openmc/mgxs_library.py

```diff
@@ -85,40 +24,37 @@ class XSdata(object):
 
     Parameters
     ----------
-    name : str, optional
+    name : str
         Name of the mgxs data set.
     energy_groups : openmc.mgxs.EnergyGroups
         Energygroup structure
     representation : {'isotropic', 'angle'}, optional
         Method used in generating the MGXS (isotropic or angle-dependent flux
         weighting). Defaults to 'isotropic'
+    temperatures : numpy.ndarray
+        Temperatures (in units of Kelvin) of the provided datasets.  Defaults
+        to a single temperature at 294K.
 
     Attributes
     ----------
     name : str
         Unique identifier for the xsdata object
-    alias : str
-        Separate unique identifier for the xsdata object
     awr : float
         Atomic weight ratio of an isotope.  That is, the ratio of the mass
         of the isotope to the mass of a single neutron.
-    kT : float
-        Temperature (in units of MeV).
+    temperatures : numpy.ndarray
+        Temperatures (in units of Kelvin) of the provided datasets.  Defaults
+        to a single temperature at 294K.
     energy_groups : openmc.mgxs.EnergyGroups
         Energy group structure
     fissionable : bool
         Whether or not this is a fissionable data set.
-    scatt_type : {'legendre', 'histogram', or 'tabular'}
+    scatter_type : {'legendre', 'histogram', or 'tabular'}
```

Comment:
Could `scatter_type` be changed to `scatter_dist` (or some other name that is a little more prescriptive)? When I think scatter type, I think (n,n)/(n,2n)/etc or elastic/inelastic.